src/Sentry/Protocol/Exceptions/SentryStackFrame.cs

```diff
@@ -248,6 +257,12 @@ public void WriteTo(Utf8JsonWriter writer)
                 writer.WriteNumber("symbol_addr", symbolAddress);
             }
 
+            // Instruction address
+            if (InstructionAddress is { } instructionAddress)
```

Comment:
This doesn't check if it's `""` or `"  "`, why did you chose this vs string.IsNullOrEmpty?